scripts/setup-db-folder.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+log_any () {
+	echo -e "[\e[$1m$2\e[0m] $3"
+}
+
+log_error   () { log_any "31" "-" "$1"; }
+log_ok      () { log_any "32" "+" "$1"; }
+log_warning () { log_any "33" "~" "$1"; }
+log_hint    () { log_any "34" "?" "$1"; }
+log_info    () { log_any "35" "i" "$1"; }
+
+check_pwd () {
+  root="$(dirname $(realpath $(dirname "$0")))"
+  [ "$root" != "$(pwd)" ] && {
```

Comment:
Are you sure you check the right path here ?

I mean, I tested the command locally and here what I found:
```bash
[ iscsc.fr ] on dev-container [?] & dirname $(realpath $(dirname .))   
/home/math/Documents

[ iscsc.fr ] on dev-container [?] & pwd     
/home/math/Documents/iscsc.fr
```
And I am right at the root of the repo. Is it because the script is in another directory `dirname` will take the directory from where the script is and not from where it is launched ? I find it utterly strange there are no other ways of checking this. And yet, I just think about doing a `cd ..` followed by a `pwd`, which is not better you could say...

Reply:
> Are you sure you check the right path here ?

yes I am! Because in fact this test is also used in `gpg-share.sh` and `bump.sh` scripts and has been introduced by #29 so it has been tested :wink:
the reason why it seems not to work in your exmaple is because the real code is
```bash
root="$(dirname $(realpath $(dirname "$0")))"
```
when you used 
```
dirname $(realpath $(dirname .))
```
However in your case `. == /home/math/Documents/iscsc.fr` when `$0 == /home/math/Documents/iscsc.fr/scripts/setup-db-folder.sh`
That explains the differences experienced!

Just try:
```
cd /home/math/Documents/iscsc.fr/scripts
./setup-db-folder.sh
```
and you'll see the error message!

Finally you're right that's kind of weird but this is currently the way we're doing it :confused: 
maybe we should:
 1. factorize some `log*` and `check_*` functions used in all `*.sh` scripts
 2. find a better way to get the parent of the current directory (because that's what we're checking here!)

Reply:
That's what I thought ! Thanks for the clarification. And for the other way, I was just wondering. This is details and we have to merge this, so don't take it into account for now